tests/mol-rendering-audio/EPUB/package.opf

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System implements Media Overlays' RS Spec "Rendering Audio" section.</dc:description>
+    <dc:identifier id="pub-id">mol-rendering-audio</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Media Overlays Tests: Rendering Audio</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://w3c.github.io/epub-specs/epub33/rs/#sec-rsconf-rendering-audio</meta>
```

Comment:
Yes I was just focusing on the MUSTs because I thought that was what we needed to tackle first. I can write test for that last SHOULD (will prob be a separate EPUB since that test does not have anything in common with the others).

And when I add the test references to the normative statement, as per step 12, I use the test IDs, but where should these be defined for this situation of having multiple tests in one EPUB?

Reply:
In that situation, you would have a single epub but multiple data-tests attributes pointing to the same test, like so:

<p id="first-anchor" data-tests="#mol-rendering-audio">First normative statement.</p>
<p id="second-anchor" data-tests="#mol-rendering-audio">Second normative statement.</p>
<p id="third-anchor" data-tests="#mol-rendering-audio">Third normative statement.</p>

And then mol-rendering-audio.epub would have three dcterms:isReferencedBy:
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#first-anchor</meta>
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#second-anchor</meta>
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#third-anchor</meta>

Make sense?

Reply:
And you can't refer to a test below the package level of the test EPUB. Is that right? 

It doesn't matter at this point for me but conceptually I am curious.

Reply:
Sorry, missed this. Yes, that's correct.